portal-web/test/functional/com/liferay/portalweb/tests/myfirsttest.testcase

```diff
@@ -0,0 +1,20 @@
+<definition component-name=<"portal-smoke">
+	<property name="testray.main.component.name" value="Smoke" />
+	
+	<command name="SuperAdminLogin" priority="5">
+		<execute function="Open" locator1="http://localhost:8080" />
+		<execute function="waitForPageToLoad" value="3000" />
```

Comment:
I don't think waitForPageToLoad is a function.  All the possible functions are found here (and in the same place in your source): https://github.com/liferay/liferay-portal/tree/master/portal-web/test/functional/com/liferay/portalweb/functions